usaspending_api/etl/management/helpers/load_submission.py

```diff
@@ -113,8 +125,15 @@ def _retrieve_and_prepare_next_chunk(self):
             from    certified_award_financial
```

Comment:
Is broker keeping this table `certified_award_financial` or may they be changing to `published_award_financial`? Same question with all other `certified_*` tables in broker.

Reply:
Well that is super unfortunate.  Yes, I believe they are creating published tables.  Oversight.

Reply:
I see from Alisa's replies to your questions that this is NOT the case.  Perhaps I knew that at one point.  I have asked them hundreds of questions over the past week.  They're probably sick of hearing from me.